app/Listeners/RecordEvent.php

```diff
@@ -2,25 +2,19 @@
 
 namespace App\Listeners;
 
+use App\Models\Activity;
 use App\Events\RecordableEvent;
 use Illuminate\Support\Facades\Auth;
 use Illuminate\Queue\InteractsWithQueue;
 use Illuminate\Contracts\Queue\ShouldQueue;
+use App\Modules\Group\Events\JudgementEvent;
```

Comment:
I don't think this is actually used in this file.

Reply:
You are correct.  Removed along with a couple of other unused deps.